clayui.com/content/docs/get-started/using-clay-in-jsps.md

```diff
@@ -0,0 +1,74 @@
+---
+title: 'Using Clay in JSPs'
+order: 6
+---
+
+<div class="nav-toc-absolute">
+<div class="nav-toc">
+
+-   [Setting up your module](#setting-up-your-module)
+-   [Using Clay taglib in your JSPs](#using-clay-taglib-in-your-jsps)
+-   [Clay components available in JSPs](#clay-components-available-in-jsps)
+
+</div>
+</div>
+
+Clay in combination with DXP provides a set of tags for creating Clay components in your JSP files.
+
+## Setting up your module
+
+To use Clay components in your JSP apps, add the following to your module:
+
+### Gradle
```

Comment:
I think we might be better off leaving out this gradle portion since this more falls under dxp specific API and the more we can focus on just the tags, the better.

Reply:
@jbalsas what do you think about this? 

My thinking is that anyone who comes looking for Clay in JSPs is gonna appreciate this information.

Reply:
I agree with @bryceosterhaus.

This is a build setup that's unique to DXP. You have customers and even Workspace posibly using Maven setups. We might move to another build system...  chances are the gradle setup already includes the dependency...

Let's stick to the JSP here.